ctx_test.go

```
@@ -5931,6 +5931,68 @@
+	require.NoError(t, err, "io.ReadAll(resp.Body)")
+	require.Equal(t, string(body), "Hello, World!")
+}
+
+// go test -run Test_Ctx_End_with_drop_middleware
+func Test_Ctx_End_with_drop_middleware(t *testing.T) {
+	app := New()
+
+	// Middleware that will drop connections
+	// that persist after c.Next()
+	app.Use(func(c Ctx) error {
+		c.Next()
```

golangci-lint (1.62.2)

Error return value of `c.Next` is not checked

(errcheck)